acquisitions/src/main/resources/thunderjet/mod-orders/features/check-estimated-price-with-composite-order.feature

```diff
@@ -0,0 +1,112 @@
+
+  Background:
+    * print karate.info.scenarioName
+
+    * url baseUrl
+
+    * call login testAdmin
+    * def okapitokenAdmin = okapitoken
+    * call login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+    * call variables
+
+
+  Scenario: Check estimated price with composite order
+    * print '## Prepare finances'
+    * configure headers = headersAdmin
+    * def histFundId = "fb7b70f1-b898-4924-a991-0e4b6312bb5f"
+    * def genrlFundId = "1b6d3338-186e-4e35-9e75-1b886b0da53e"
+    * def miscHistFundId = "a89eccf0-57a6-495e-898d-32b9b2210f2f"
+    * def histBudgetId = call uuid
+    * def genrlBudgetId = call uuid
+    * def miscHistBudgetId = call uuid
+    * def v = call createFund { id: "#(histFundId)" }
+    * def v = call createBudget { id: "#(histBudgetId)", fundId: "#(histFundId)", allocated: 1000 }
+    * def v = call createFund { id: "#(genrlFundId)" }
+    * def v = call createBudget { id: "#(genrlBudgetId)", fundId: "#(genrlFundId)", allocated: 1000 }
+    * def v = call createFund { id: "#(miscHistFundId)" }
+    * def v = call createBudget { id: "#(miscHistBudgetId)", fundId: "#(miscHistFundId)", allocated: 1000 }
+    * configure headers = headersUser
+
+    * print '## Prepare the order'
+    * def po = read('classpath:samples/mod-orders/compositeOrders/po-listed-print-monograph.json')
+    # Make sure expected number of PO Lines available
+    * assert po.compositePoLines.length == 2
+    * assert po.compositePoLines[0].orderFormat == "P/E Mix"
+    * assert po.compositePoLines[1].orderFormat == "Electronic Resource"
+
+    # Prepare cost details for the first PO Line (see MODORDERS-180 and MODORDERS-181)
+    * def cost = po.compositePoLines[0].cost
+    * set cost.additionalCost = 10.0
+    * set cost.discount = 3.0
+    * set cost.discountType = "percentage"
+    * set cost.quantityElectronic = 1
+    * set cost.listUnitPriceElectronic = 5.5
+    * set cost.quantityPhysical = 3
+    * set cost.listUnitPrice = 9.99
+    * set cost.poLineEstimatedPrice = null
+    * def expectedTotalPoLine1 = 44.41
+
+    # Prepare cost details for the second PO Line (see MODORDERS-180 and MODORDERS-181)
+    * def cost = po.compositePoLines[1].cost
+    * set cost.additionalCost = 2
+    * set cost.discount = 4.99
+    * set cost.discountType = "amount"
+    * set cost.quantityElectronic = 3
+    * set cost.listUnitPriceElectronic = 11.99
+    * set cost.poLineEstimatedPrice = null
+    * def expectedTotalPoLine2 = 32.98
+
+    * print '## Create the composite order'
+    Given path 'orders/composite-orders'
+    And request po
+    When method POST
+    Then status 201
+
+    * print '## Check the returned composite order'
```

Comment:
I often look for the output for a particular part of the scenario. For this I simply copy and find the text, so I need it to be printed. The endpoints alone are not sufficient, because the same endpoint can be called several times in the same scenario. The "##" at the beginning (which I am using for the first time) is also helping to distinguish with the rest of the execution.